package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "maxmind",
-  "version": "2.5.0",
+  "version": "2.6.0",
```

Comment:
Please revert it - normally I bump version during release - see `./scripts/release` file for details.